msi/GitProduct.wxs

```diff
@@ -1,78 +1,283 @@
-<?xml version='1.0'?>
```

Comment:
Can someone help me by telling me how I might go about fixing the line-ending whitespace errors? Stupid me didn't have the `core.autocrlf` setting in my global git config settings.....